astroquery/cadc/core.py

```diff
@@ -428,7 +428,9 @@ def get_image_list(self, query_result, coordinates, radius):
                 '{}?{}'.format(self.data_link_url,
                                urlencode({'ID': pid_sublist}, True)))
             for service_def in datalink.bysemantics('#cutout'):
-                access_url = service_def.access_url.decode('ascii')
+                access_url = service_def.access_url
+                if isinstance(access_url, bytes):  # ASTROPY_LT_4_1
```

Comment:
Can't compare using `ASTROPY_LT_4_1` directly because of the mock stuff in its tests.